src/epcpm/cli/main.py

```diff
@@ -111,6 +112,50 @@ def build(
     click.echo("done")
 
 
+@export.command()
+@epcpm.cli.utils.project_option(required=True)
+@epcpm.cli.utils.target_path_option(required=True)
+@epcpm.cli.utils.pmvs_overlay_recipes_path_option(required=True)
+@click.option("--generate-formatted-output", "generate_formatted_output", is_flag=True)
+def docs(
+    project: str,
+    target_path: str,
+    pmvs_overlay_recipes_path: str,
+    generate_formatted_output: bool,
+) -> None:
+    """
+    Export PM documentation to embedded project directory
+
+    Args:
+        project: path to PM project file
+        target_path: path to root target directory
+        pmvs_overlay_recipes_path: path to PMVS overlay recipes directory (contains base.json)
+        generate_formatted_output: generate formatted output of the documentation (takes a long time)
+    Returns:
+
+    """
+    project = pathlib.Path(project)
+    target_path = pathlib.Path(target_path)
+
+    pmvs_base = pathlib.Path(pmvs_overlay_recipes_path) / "base.json"
+    pmvs_configuration = epyqlib.pm.valueset.OverlayConfiguration.load(pmvs_base)
+    pmvs_output_path = pmvs_configuration.reference_output_path()
+
+    paths = epcpm.importexportdialog.paths_from_directory(target_path)
+
+    loaded_project = epcpm.project.loadp(project)
+
+    epcpm.importexport.generate_docs(
+        project=loaded_project,
+        pmvs_path=pmvs_output_path,
+        paths=paths,
+        generate_formatted_output=generate_formatted_output,
+    )
+
+    click.echo()
+    click.echo("done")
```

Comment:
Remove echos?

Reply:
I was following the lead of a couple other CLI commands (in the same file). I never liked the "done" message, so I changed it. b35976bd0f34800986948d333ce3f03d6b6141f4